app/controllers/customers_controller.rb

```diff
@@ -0,0 +1,6 @@
+class CustomersController < ApplicationController 
+  def show
+    @customer = Customer.find(params[:id])
+    @customer_items = @customer.items
```

Comment:
I would probably just call `@customer.items` in your view vs creating this `@customer_items` instance variable. Keep you controller lean.